com.bkmarriott.coupon/src/main/java/com/bkmarriott/coupon/infrastructure/persistence/entity/MemberCouponEntity.java

```diff
@@ -0,0 +1,66 @@
+package com.bkmarriott.coupon.infrastructure.persistence.entity;
+
+import com.bkmarriott.coupon.domain.MemberCoupon;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Table(name = "m_member_coupon")
+@Entity(name = "MemberCoupon")
+public class MemberCouponEntity {
```

Comment:
```suggestion
public class MemberCouponEntity extends BaseEntity {
```
혹시 여기만 auditing을 제외시킨 다른 이유가 있나요?